api-catalog-ui/frontend/src/components/Wizard/WizardDialog.jsx

```diff
@@ -76,6 +80,130 @@ export default class WizardDialog extends Component {
         }
     };
 
+    /**
```

Comment:
the wizard dialog needs additional unit tests to cover the new code, as reported by the coverage analysis